Formula/proj.rb

```diff
@@ -33,9 +33,16 @@ class Proj < Formula
     sha256 "b9838ae7e5f27ee732fb0bfed618f85b36e8bb56d7afb287d506338e9f33861e"
   end
 
+  # Fixes https://github.com/OSGeo/PROJ/issues/2064 using https://github.com/OSGeo/PROJ/pull/2067
```

Comment:
Can you write a comment like, `# Remove after version > 7.0.0`?

Reply:
Done!